metricbeat/helper/socket/listeners.go

```diff
@@ -36,10 +36,18 @@ const (
 	Listening
 )
 
+// Names for the direction of a connection
+const (
+	// XXX: Rephrase this to inbound/outbound to match ECS recommendations?
+	IncomingName  = "incoming"
+	OutgoingName  = "outgoing"
+	ListeningName = "listening"
```

Comment:
Should we change to the names recommended by ECS?

We could keep old and new ones in the old and new fields.

Reply:
If we keep the both values around, we can't easily disable it when `migration.enabled: true` is set. It's possible but would require quite a bit of work. Because of this I suggest we make this a breaking change and don't keep the old version around.

Reply:
@andrewkroh One more place where we could use the constants from ECS in Go code ;-)

Reply:
Ok, I'll change this and add a note about the breaking change.